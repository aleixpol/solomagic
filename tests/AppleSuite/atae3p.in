\ref something0
\mt somethingsomething ataendi somethingsomething

\ref something1
\mt somethingsomething korino somethingsomething

