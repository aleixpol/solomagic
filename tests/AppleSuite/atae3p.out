\ref something0
\mt somethingsomething ataendi somethingsomething
\ma somethingsomething ataengri somethingsomething

\ref something1
\mt somethingsomething korino somethingsomething
\ma somethingsomething korino somethingsomething
